modules/ROOT/pages/test-recorder.adoc

```diff
@@ -0,0 +1,15 @@
+= Test Recorder in Studio (Beta)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The test recorder enables you to record a processing flow and then configure a unit test based on the captured event.
+
+The test recorder is an open beta feature and is compatible with Anypoint Studio 7.x.x and later, and MUnit 2.2.1 and later.
```

Comment:
This is not correct. At the very least, we need MUnit 2.2.2 and Mule 4.2.2, and I haven't checked the recorder with any Studio version older than 7.4.0. Later I will do some testing and send you which versions should be placed here.

Reply:
Also, you need a patched version of 4.2.2, so you will need to pull the patch from the updatesite that has the required dependencies on it. The users should be prompted as soon as the new version becomes available and they might not even realise that they have already fetched the recorder dependencies, but we should still notify them just in case there is any issues.

Reply:
Now that we are nearing release supported versions have already been defined. It will be needing at least MUnit 2.2.4 (which comes bundled with MUnit's Studio plugin, so this should be transparent to the user), the patched Mule 4.2.2, and Studio 7.4.0.

Reply:
Also, it might be good to note that these versions will definitely change from BETA to GA.